style.css

```diff
@@ -743,3 +747,680 @@ button:active {
   background: #0416b6;
   color: white;
 }
+
+@media screen and (min-width: 768px) {
```

Comment:
- [ ] **[Optional]** I highly recommend fixing this (refer to the image attached) grey space.

![image](https://user-images.githubusercontent.com/92215878/186732193-7011a6ba-0758-4002-997d-0983a723658f.png)